apps/examples/log_dump/log_dump_main.c

```diff
@@ -47,32 +47,39 @@ int log_dump_main(int argc, char *argv[])
 	int fd = OPEN_LOGDUMP();
 
 	if (fd < 0) {
-		printf("Please ensure that procfs is mounted\n");
+		printf("log_dump_main : Please ensure that procfs is mounted\n");
 		return -1;
 	}
 
 	/* Log dump starts automatically from boot up.
 	 * To test start, intentionally add to stop. */
 	if (STOP_LOGDUMP_SAVE(fd) < 0) {
-		printf("Failed to stop log dump, errno %d\n", get_errno());
+		printf("log_dump_main : Failed to stop log dump, errno %d\n", get_errno());
 		goto errout;
 	}
 
-	printf("This Log Should NOT be saved!!!\n");
+	if (argc > 1) {
+		if (SET_LOGDUMP_FETCH_BLOCKS(fd, argv[1])) {
```

Comment:
There is no help option. Please add it to know how to use. It should include how to get all nodes and how to get partial node from the last.

Reply:
Done, added usage.